Drop-In/src/main/java/com/braintreepayments/api/BottomSheetFragment.java

```diff
@@ -51,7 +51,14 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
 
         View view = inflater.inflate(R.layout.bt_fragment_bottom_sheet, container, false);
         backgroundView = view.findViewById(R.id.background);
+
         viewPager = view.findViewById(R.id.view_pager);
+        // TODO: investigate view pager saveInstanceState restoration
+        viewPager.setSaveEnabled(false);
```

Comment:
It might make sense to tackle this TODO in this PR too if possible. I think we need to save instance state of the viewPager or if you rotate the screen while in the vault manager screen, it jumps back to the payment method page instead.

Reply:
Ok yeah that makes sense